rate_counter/rate/tasks.py

```diff
@@ -96,3 +104,37 @@ def get_url(inst='1', group='', semestr='1', semestr_part='1'):
                                                                                                 semestr,
                                                                                                 semestr_part,
                                                                                                 )
+
+
+def rate_graphs(id):
+    '''
+    prepare data for json column charts
```

Comment:
Why this functions are in 'tasks' module? tasks module is for periodic tasks not for functions.